administrator/components/com_guidedtours/tmpl/tours/default.php

```diff
@@ -0,0 +1,271 @@
+                                ?>
+                                <input type="text" class="hidden text-area-order"
+                                       name="order[]" size="5" value="<?php echo $item->ordering; ?>">
+                            <?php endif; ?>
+                        </td>
+
+                        <!-- Item State -->
+                        <td class="text-center">
+                            <?php echo HTMLHelper::_(
+                                'jgrid.published',
+                                $item->published,
+                                $i,
+                                'tours.',
+                                $canChange
+                            ); ?>
+                        </td>
+
+                        <th scope="row" class="has-context">
+                            <div>
+                                <?php if ($canEdit) : ?>
+                                    <a href="<?php echo Route::_('index.php?option=com_guidedtours&task=tour.edit&id=' . $item->id); ?>" title="<?php echo Text::_('JACTION_EDIT'); ?> <?php echo $this->escape($item->title); ?>">
+                                        <?php echo $this->escape($item->title); ?>
+                                    </a>
+                                <?php else : ?>
+                                    <?php echo $this->escape($item->title); ?>
+                                <?php endif; ?>
+                                <div class="small break-word">
+                                    <?php if ($item->note) : ?>
+                                        <?php echo Text::sprintf('JGLOBAL_LIST_NOTE', $this->escape($item->note)); ?>
+                                    <?php endif; ?>
+                                </div>
+                            </div>
+                        </th>
+
+                        <td class="">
+                            <?php echo StringHelper::truncate($item->description, 200, true, false); ?>
+                        </td>
+
+                        <!-- Adds access labels -->
+                        <td class="small text-center d-none d-md-table-cell">
+                            <?php
+
+                            if ($this->escape($item->access) == "1") {
+                                echo Text::_('COM_GUIDEDTOURS_ACCESS_PUBLIC');
+                            } elseif ($this->escape($item->access) == "2") {
+                                echo Text::_('COM_GUIDEDTOURS_ACCESS_REGISTERED');
+                            } elseif ($this->escape($item->access) == "3") {
+                                echo Text::_('COM_GUIDEDTOURS_ACCESS_SPECIAL');
+                            } elseif ($this->escape($item->access) == "6") {
+                                echo Text::_('COM_GUIDEDTOURS_ACCESS_SUPER_USERS');
+                            } else {
+                                echo Text::_('COM_GUIDEDTOURS_ACCESS_GUEST');
+                            }
```

Comment:
Please look at all other components for the correct way to do this. Thse access levels are not static

Reply:
Definitely. It will be fixed shortly.